.travis.yml

```diff
@@ -11,9 +11,11 @@ before_install:
     - 'if [[ $GROUP != js* ]]; then COVERAGE=""; fi'
 install:
     - pip install -f travis-wheels/wheelhouse -r requirements.txt -e file://$PWD#egg=ipython[test] coveralls
+    - pip install codecov
```

Comment:
I removed it, it was there mostly so that I can track the increase in coverage on codecov website. 
On the same line it makes pip crash (cf other PR)